Parsers/OSSEC/OSSECEvent.txt

```diff
@@ -0,0 +1,52 @@
+// Usage Instruction : 
+// Paste below query in log analytics, click on Save button and select as Function from drop down by specifying function name and alias as OSSECEvent.
+// Function usually takes 10-15 minutes to activate. You can then use function alias from any other queries (e.g. OSSECEvent | take 10).
+// Reference : Using functions in Azure monitor log queries : https://docs.microsoft.com/azure/azure-monitor/log-query/functions
+CommonSecurityLog 
+| where DeviceVendor has 'OSSEC'
+| extend EventVendor = DeviceVendor
+| extend EventProduct = DeviceProduct
+| extend EventProductVersion = DeviceVersion
+| extend RuleName = Activity
+| extend RuleNumber = DeviceEventClassID
+| extend Classification = extract(@'(classification|cat)=(\s)?(.*?)(,$|;)', 3, AdditionalExtensions)
+| extend DvcHostname = DeviceAddress
+| extend EventSeverity = LogSeverity
+| extend SrcIpAddr = SourceIP
+| extend SrcHostname = SourceHostName
+| extend SrcUserName = SourceUserName
+| extend EventMessage = Message
+| extend FilePath = FileName
+| extend Location = DeviceCustomString1
+| extend OldMd5 = DeviceCustomString2
+| extend NewMd5 = DeviceCustomString3
+| project TimeGenerated
+        , EventVendor
+        , EventProduct
+        , EventProductVersion
+        , RuleName
+        , RuleNumber
+        , Classification
+        , DvcHostname
+        , EventSeverity
+        , SrcIpAddr
+        , SrcHostname

```

Comment:
Should it not be 'SrcDvcHostname'? Please refer https://docs.microsoft.com/en-us/azure/sentinel/normalization-schema for field names.

Reply:
Here we referring the device name of the device generating the message which is DvcHostname  according to https://docs.microsoft.com/en-us/azure/sentinel/normalization-schema